lib/charms/mysql/v0/mysql.py

```diff
@@ -1401,9 +1397,13 @@ def execute_backup_commands(
             --storage=s3
             --parallel=10
             --md5
-            --s3-bucket={s3_bucket}
-            --s3-endpoint={s3_endpoint}
-            {s3_directory}
+            --storage=S3
```

Comment:
this option is already specified on line 1397

Reply:
Fixed in https://github.com/canonical/mysql-operator/pull/204/commits/ff9bec18e87b75e09280c1ad93877d5efbc5a31f